backend/utils/paypal.js

```diff
@@ -206,10 +218,72 @@ const verifySignMiddleware = async (req, res, next) => {
 
   next()
 }
+
+/**
```

Comment:
We may want to move this to logic/payments/paypal and for stripe under logic/payments/stripe